src/main/java/io/nats/service/Service.java

```diff
@@ -63,32 +63,26 @@ public class Service {
         dInternals = new ArrayList<>();
         startStopLock = new ReentrantLock();
 
+        // build responses first. info needs to be available when adding service endpoints.
+        pingResponse = new PingResponse(id, b.name, b.version, b.metadata);
+        infoResponse = new InfoResponse(id, b.name, b.version, b.metadata, b.description);
```

Comment:
build responses first. info needs to be available when adding service endpoints.